.github/workflows/pr-build.yml

```diff
@@ -80,6 +80,10 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: yarn patch-project
```

Comment:
I don't like this approach.
First of all, you forgot to add it to `release.yml`
Secondly, it makes project maintenance more complicated (and for such specifics obviously we need to add some documentation and keep it updated).

So, I don't like the idea of patching sentry.
However, if we really need it — I'd prefer to do it in `postinstall` or as a part of `yarn package-*`. Then we will have the same behavior everywhere. No surprises.